runelite-client/src/main/java/net/runelite/client/plugins/wikilookup/WikiOrbInterface.java

```diff
@@ -0,0 +1,156 @@
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package net.runelite.client.plugins.wikilookup;
+
+import java.awt.Rectangle;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Client;
+import net.runelite.api.ScriptID;
+import net.runelite.api.SpriteID;
+import net.runelite.api.WidgetType;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.callback.ClientThread;
+import net.runelite.client.config.ConfigManager;
+
+@Slf4j
+@Singleton
+public class WikiOrbInterface
+{
+	private static final String WIKI_ORB = "Wiki Lookup";
+	private static final int TAB_HEIGHT = 40;
+	private static final int BUTTON_HEIGHT = 20;
+	private static final int MARGIN = 1;
+	private final Client client;
+	private final ClientThread clientThread;
+	private final ConfigManager configManager;
+	private final WikiLookupConfig config;
+	private final Rectangle bounds = new Rectangle();
+	private boolean loaded = false;
+	private boolean turnedOff = true;
+	
+	@Getter
+	private Widget wikiOrb;
+	@Getter
+	private Widget orbFrame;
+	@Getter
+	private Widget orbBackground;
+	@Getter
+	private Widget orbBackgroundActivated;
+
+	@Getter
+	private Widget parent;
+	
+	@Inject
+	private WikiOrbInterface(
+			final Client client,
+			final ClientThread clientThread,
+			final ConfigManager configManager,
+			final WikiLookupConfig config)
+	{
+		this.client = client;
+		this.clientThread = clientThread;
+		this.configManager = configManager;
+		this.config = config;
+	}
+	
+	public void init()
+	{
+		if (isHidden())
+		{
+			return;
+		}
+		loaded = true;
+		parent = client.getWidget(WidgetInfo.MINIMAP_ORBS);
+		wikiOrb = createGraphic("", SpriteID.MINIMAP_MAP_ORB_FRAME, -1, 30 , 30 , 170, 0, true);
+		orbBackground = createGraphic("", SpriteID.MINIMAP_ORB_RUN, -1, 25 , 25 , 173, 3, false);
+		orbBackgroundActivated = createGraphic("", SpriteID.MINIMAP_ORB_RUN_ACTIVATED, -1, 25 , 25 , 173, 3, false);
+		orbBackgroundActivated.setHidden(true);
+		orbFrame = createGraphic("", SpriteID.BANK_SEARCH, -1, 20, 20, 174, 3, false);
```

Comment:
Your sprite icon looks squished in the photo. I would also make these numbers less magic and calculate based on parent width/height.

Reply:
I have little to no experience in that regard. Mind helping me? Or showing me where its been done before so I can glean?

Reply:
Use the widget inspector and change the size until it looks good